clients/stellar-relay-lib/src/node/mod.rs

```diff
@@ -19,6 +19,19 @@ pub struct NodeInfo {
 	pub network_id: NetworkId,
 }
 
+impl NodeInfo {
+	pub(crate) fn new(cfg:&NodeInfoCfg) -> Self {
```

Comment:
Eliminate an all out clone of `NodeInfoCfg`. We only need to clone 1 field, the `version_str`